.pre-commit-config.yaml

```diff
@@ -37,6 +37,6 @@
     files: ".*/static2/.*graphql$"
   - id: flake8
     name: flake8
-    entry: bash -c "PATH=./venv/bin/:$PATH flake8 `git diff --name-only --diff-filter=d --cached -- '*.py'|grep assembl/`"
+    entry: bash -c "PATH=./venv/bin/:$PATH flake8 `git diff --name-only --diff-filter=d --cached -- '*.py'`"
```

Comment:
The grep here was previously "grep assembl/graphql/" and has been transformed to just "grep assembl/" when we did the flake8 branch with the exclude files. So it is not necessary anymore, and the issue here was that because of the grep, modifing setup.py or fabfile.py was executing just `fake8` in the pre-commit, without any args, so it tried to check the working directory including venv and taking 30 minutes or more to execute.